pyaedt/misc/Jupyter.py_build

```diff
@@ -0,0 +1,48 @@
+"""
+* * * This script is meant to run in IronPython within the Ansys Electronics Desktop * * *
+
+It looks for a reference to a python interpreter in python_interpreter.bat
+
+This script executes the "hidden" CPython script console_setup (hidden because the .py was omitted)
+This hidden script sets up the hfss variable based on the command line arguments and kicks off the interactive python
+shell.
+
+"""
+
+
+import os
+import subprocess
+import shutil
+import random
+import string
+#choose file to launch
+
+def generate_unique_name(rootname, suffix="", n=6):
+
+    char_set = string.ascii_uppercase + string.digits
+    uName = "".join(random.choice(char_set) for _ in range(n))
+    unique_name = rootname + "_" + uName
+    if suffix:
+        unique_name += suffix
+    return unique_name
+
+#launch file
+proj_dir = oDesktop.GetProjectDirectory()
+os.chdir(proj_dir)
+
+pyaedt_toolkit_dir = r'''C:\Program Files\AnsysEM\v222\Win64\syslib\Toolkits\HFSS\PyAEDT'''
+jupyter_exe = r'''C:\Users\mcapodif\AppData\Roaming\pyaedt_env_ide\v222\Scripts\jupyter.exe'''
```

Comment:
I am guessing, is it expected to be hard-coded? Just to be sure.

Reply:
you are right

Reply:
I'm changing right now. Great catch